src/main/java/org/takes/rq/RqMultipart.java

```diff
@@ -506,6 +491,61 @@ private static StringBuilder fakeBody(final Request... dispositions)
                 .append(String.format("--%s--", BOUNDARY));
             return builder;
         }
+
+        /**
+         * This class is  for the specific implementation
+         * for the RqMultipart. see the overridden methods for details.
+         *
+         * @todo #517:30min re-design the class.
+         *  Why the super class is encapsulated inside the class?
+         *  Find an implementation without doing aggregation
+         *  and extending the same class
+         */
+        private static class RqMultipartRequest implements Request {
+
+            /**
+             * Request object holds value for the header.
```

Comment:
@sebing this are rather two sentences: `Request object. Holds value of the header.`